SPECS-EXTENDED/wsmancli/wsmancli.spec

```
@@ -2,12 +2,10 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Name:           wsmancli
 Version:        2.6.0
-Release:        12%{?dist}
+Release:        13%{?dist}
 License:        BSD
 Url:            http://www.openwsman.org/
-# You can get this tarball here:
-# https://github.com/Openwsman/wsmancli/archive/v%{version}.tar.gz
-Source:         wsmancli-%{version}.tar.gz
+Source0:        https://github.com/Openwsman/wsmancli/archive/v%{version}.tar.gz
```

You accidentally changed the source tarball's name:

```suggestion
Source0:        https://github.com/Openwsman/wsmancli/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```